Sources/ArcGISToolkit/Components/Offline/OfflineMapAreasView.swift

```diff
@@ -89,18 +102,6 @@ public struct OfflineMapAreasView: View {
             if !models.isEmpty {
                 List(models) { preplannedMapModel in
                     PreplannedListItemView(model: preplannedMapModel, selectedMap: $selectedMap)
-                        .onDownload {
-                            guard let portalItem else { return }
-                            OfflineManager.shared.saveMapInfo(for: portalItem)
-                        }
-                        .onRemoveDownload {
-                            guard let portalItemID = portalItem?.id,
-                                  models.filter(\.status.isDownloaded).isEmpty else { return }
-                            OfflineManager.shared.deleteMapInfo(for: portalItemID)
-                        }
-                        .onChange(of: selectedMap) { _ in
-                            dismiss()
```

Comment:
dismiss can happen when we set it in the child view, no need for the onChange modifier here